daemon/cmd/handlers/endpoints.go

```diff
@@ -0,0 +1,288 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package handlers
```

Comment:
does this need to live under `daemon/cmd`? maybe just `daemon/handlers` or `daemon/api/handlers` or something?

Reply:
yeah good point. `daemon/restapi` could work. `handlers` is too vague.